File: Bits/inverserBinaire.js
/**
 * Inverser la conversion binaire d'un nombre entier
 * @param {number} int - Nombre entier à convertir en binaire, puis à inverser.
 * @return {number} Nombre binaire inversé
 */
const inverserBinaire = int => parseInt(int.toString(2).split('').reverse().join(''), 2);

// Test
console.log(inverserBinaire(8)); // 1000 => 0001, soit 1

File: Python/Bits/inverserBinaire.py
def inverserBinaire(n: int) -> int:
    """
    Description:
        Inverse la conversion binaire d'un nombre entier.
    
    Paramètres:
        n: {int} -- Le nombre à convertir, puis à inverser.
    
    Retourne:
        {int} -- Le nombre binaire inversé.
    
    Exemple:
        >>> inverserBinaire(8)
        1

        1000 = 8, donc 0001 = 1inverserBinaire.py
    """
    return int(bin(n)[2:][::-1], 2)